<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectDialog</class>
 <widget class="QDialog" name="SelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="imgPreview">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>160</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>160</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Preview</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="rscr.qrc">:/resource/splash.bmp</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="imgTitleScreen">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>160</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>160</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Title screen</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="rscr.qrc">:/resource/splash.bmp</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="tvDrivers">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Title</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Rom</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Game Info</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leGameInfo">
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>ROM Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leRomName">
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>ROM Info</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="leRomInfo">
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Released by</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="leReleasedBy">
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Genre</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="leGenre">
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Notes</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="leNotes">
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lblCounter">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnRomInfo">
         <property name="text">
          <string>Rom Info</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="ckbShowAvaliable">
            <property name="text">
             <string>Show avaliable</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckbShowUnavaliable">
            <property name="text">
             <string>Show unavaliable</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckbShowClones">
            <property name="text">
             <string>Always show clones</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckbUseZipnames">
            <property name="text">
             <string>Use zipnames</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ckbLatinTextOnly">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Latin text only</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="btnRomDirs">
              <property name="text">
               <string>ROMs Dirs...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnScanRoms">
              <property name="text">
               <string>Scan ROMs</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="tvFilters">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string>Filters</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Filters</string>
        </property>
        <property name="checkState">
         <enum>Unchecked</enum>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
        </property>
        <item>
         <property name="text">
          <string>BoardType</string>
         </property>
         <property name="checkState">
          <enum>Unchecked</enum>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
         </property>
         <item>
          <property name="text">
           <string>Genuine</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bootleg</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Demo</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hack</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Homebrew</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Prototype</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Family</string>
         </property>
         <property name="checkState">
          <enum>Unchecked</enum>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Genre</string>
         </property>
         <property name="checkState">
          <enum>Unchecked</enum>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hardware</string>
         </property>
         <property name="checkState">
          <enum>Unchecked</enum>
         </property>
         <item>
          <property name="text">
           <string>Capcom (Other)</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cave</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CPS-1</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CPS-2</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CPS-3</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Data East</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Galaxian</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Irem</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Kaneko</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Konami</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Neo Geo</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pacman</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PGM</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Psikyo</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sega</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Seta</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Taito</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Technos</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Toaplan</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Misc (pre 90s)</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Misc (post 90s)</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Master System</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Megadrive</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>PC-Engine</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SNES</string>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
         </item>
        </item>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="leSearch">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSearch">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPlay">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rscr.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>SelectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>555</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
